static/src/stylesheets/layout/_footer.scss

```diff
@@ -67,14 +67,19 @@ $c-primary-footer-background-side-bar: mix($c-primary-footer-background, #ffffff
 .l-footer__secondary {
     box-sizing: border-box;
     @include fs-textSans(1);
-    padding-right: $gs-gutter * .5;
-    padding-left: $gs-gutter * .5;
+    padding-right: $gs-gutter / 2;
+    padding-left: $gs-gutter / 2;
 
     @include mq(mobileLandscape) {
         padding-right: $gs-gutter;
         padding-left: $gs-gutter;
     }
 
+    @include mq($from: tablet, $until: desktop) {
```

Comment:
this mq seems redundant since it contains the same rules as the parent selector. if `mobileLandscape` is the only place it should differ, you can rid of it and say `$from: mobileLandscape, $until: phablet` for the other one